js/default.js

```diff
@@ -43,6 +44,47 @@ if (window.platformType === 'windows') {
   })
 }
 
+// ISSUE 753: Someone wants to right click on address bar to copy/paste/stuff
+InputMenu = window.Menu.buildFromTemplate([{
```

Comment:
This is going to send a synchronous IPC to the main process, which could potentially slow down startup, so it might be better to delay creating the menu until the contextmenu event actually happens.